src/Doctrine/Orm/Filter/FreeTextQueryFilter.php

```diff
@@ -46,14 +47,26 @@ public function apply(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $q
         }
 
         $parameter = $context['parameter'];
+        $qb = clone $queryBuilder;
+        $qb->resetDQLPart('where');
+        $qb->setParameters(new ArrayCollection());
         foreach ($this->properties ?? $parameter->getProperties() ?? [] as $property) {
             $this->filter->apply(
-                $queryBuilder,
+                $qb,
                 $queryNameGenerator,
                 $resourceClass,
                 $operation,
                 ['parameter' => $parameter->withProperty($property)] + $context
             );
         }
+
+        $queryBuilder->andWhere($qb->getDQLPart('where'));
+        $parameters = $queryBuilder->getParameters();
+
+        foreach ($qb->getParameters() as $p) {
+            $parameters->add($p);
+        }
+
+        $queryBuilder->setParameters($parameters);
```

Comment:
I think this will remove all previous parameters, for example, from an extension. A safer approach would be to use `$queryBuilder->getParameters()->add()`. At least that was what I did while playing with a workaround for this issue.

Reply:
oh, yes, you are right. Row 64. Sorry for false alarm